src/visitor/FoldConstantExprVisitor.cpp

```diff
@@ -75,6 +77,14 @@ void FoldConstantExprVisitor::visit(FunctionCallExpression *expr) {
             }
         }
     }
+    auto result = FunctionManager::getStateless(*expr->name(), expr->args()->args().size());
```

Comment:
prefer `sideEffect` or `isPure` than `stateless` when we talk about function

Reply:
> prefer `sideEffect` or `isPure` than `stateless` when we talk about function

Agreed. `isPure` is better.

Reply:
Ok, fixed.